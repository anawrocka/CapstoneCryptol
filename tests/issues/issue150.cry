maxSeq' xs = ys
      where ys = [0] # [ max x  y | x <- xs
                                  | y <- ys
                       ]
