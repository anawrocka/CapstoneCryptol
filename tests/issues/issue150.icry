:l issue150.cry
:t maxSeq'
