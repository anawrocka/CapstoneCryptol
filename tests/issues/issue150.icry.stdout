Loading module Cryptol
Loading module Cryptol
Loading module Main
maxSeq' : {a, b} (fin a) => [b][a] -> [1 + b][a]
